ontology/cdno.md

```diff
@@ -29,6 +29,11 @@ tracker: https://github.com/Southern-Cross-Plant-Science/cdno/issues
 license:
   url: https://creativecommons.org/licenses/by/3.0/
   label: CC BY 3.0
+  
+homepage: https://cdno.info/
```

Comment:
note that there's already a homepage key in this configuration - you should override it rather than duplicating it